staging/src/k8s.io/cloud-provider/controllers/service/controller.go

```diff
@@ -56,12 +56,12 @@ const (
 	minRetryDelay = 5 * time.Second
 	maxRetryDelay = 300 * time.Second
 
-	// labelNodeRoleExcludeBalancer specifies that the node should not be considered as a target
+	// labelNodeExcludeBalancers specifies that the node should not be considered as a target
 	// for external load-balancers which use nodes as a second hop (e.g. many cloud LBs which only
 	// understand nodes). For services that use externalTrafficPolicy=Local, this may mean that
 	// any backends on excluded nodes are not reachable by those external load-balancers.
 	// Implementations of this exclusion may vary based on provider.
-	labelNodeRoleExcludeBalancer = "node.kubernetes.io/exclude-from-external-load-balancers"
+	labelNodeExcludeBalancers = "node.kubernetes.io/exclude-from-external-load-balancers"
```

Comment:
This seems fine, but I wonder at this point if we should move this constant to somewhere like https://github.com/kubernetes/api/blob/master/core/v1/well_known_labels.go since it's now stable

Reply:
Thanks for the review @andrewsykim! I'll do as you suggested :)